isis/src/base/objs/PvlKeyword/PvlKeyword.cpp

```diff
@@ -1391,8 +1393,24 @@ namespace Isis {
           KEYWORD
 
         Well, no value/units may exist so we're done processing this keyword.
+        Now check if it's a reserved keyword that is valueless
        */
-      return 1; // Valid & Successful
+      bool allowedValueless = false;
+      std::vector<QString> reservedKeywordNames = {"OBJECT",
```

Comment:
You might consider declaring this std::vector as `static const` for efficiency gains.